names.txt

```diff
@@ -95,4 +95,5 @@ Sikiru Rashidat
 Patrick Aziken
 Umeh Cassandra
 Medina Folashade Adeniran
-Kingsley Umujeyan
\ No newline at end of file
+Kingsley Umujeyan
```

Comment:
let me use here and practice comment abeg

Reply:
Yes